packages/core/src/Modifiers.ts

```diff
@@ -268,7 +268,7 @@ export class Modifiers {
      * @param callbackfn
      */
     some(callbackfn: (value: Modifier, index: number, array: Modifier[]) => unknown): boolean {
-        return this._contents?.some(callbackfn);
+        return this._contents?.some(callbackfn) || false;
```

Comment:
@sebgeelen My bad: `?.` actually returns `undefined` when it fails rather than `false`. Here is a particular case where the difference matter. cc @Zinston